en/tutorials-and-examples/blog/part-two.rst

```diff
@@ -195,6 +195,7 @@ ArticlesController now::
         public function index()
         {
              $this->set('articles', $this->Articles->find('all'));
+             $this->set(compact('articles'));
```

Comment:
This will actually cause the code to not work, as there is no `$articles` variable in the local scope.